packages/reports/addon-test-support/vertical-collection.js

```diff
@@ -0,0 +1,86 @@
+import { assert } from '@ember/debug';
+import { getContext } from '@ember/test-helpers';
+import { set } from '@ember/object';
+
+const verticalCollectionKey = 'component:vertical-collection';
+
+/**
+ * Checks whether a component is registered as a vertical-collection
+ * @param {Component} component - The component instance to check
+ * @returns {Boolean} - true if the component is a vertical collection
+ */
+function isVerticalCollection(component) {
+  return component && component._debugContainerKey === verticalCollectionKey;
+}
+
+/**
+ * Finds a vertical collection visible under the given selector, the selector must pin down a single instance
+ * @param {String} selector - the query selector to pin down the vertical collection
+ * @returns {Component} - the vertical collection component visible under the given selector
+ */
+export function getVerticalCollection(selector = 'body') {
+  const { owner } = getContext();
+  assert('getVerticalCollection called with no owner', owner);
+
+  const componentByView = owner.lookup('-view-registry:main');
+  if (selector.startsWith('ember')) {
```

Comment:
Are we checking for an id selector?

Reply:
Yea, looking for something like `ember1`

Reply:
@jkusa maybe just replace with something simple like
```suggestion
  if (selector in componentByView) {
```

Reply:
If expecting an `id` selector, shouldn't we expect something like this `#ember526`?